MultiSlider.js

```diff
@@ -70,6 +74,10 @@ export default class MultiSlider extends React.Component {
       slipDisplacement: 200,
     },
     customMarker: DefaultMarker,
+
+      customMarkerLeft: DefaultMarker,
```

Comment:
wrong indentation here

Reply:
Updated in last commit:
https://github.com/sadeghipour/react-native-multi-slider/blob/master/MultiSlider.js#L78